### PR TITLE
Ledmanager for rpi4

### PR DESCRIPTION
**Pull request by giggsoff**
Support for onboard LED control from ledmanager for RPI4.

### CODE REVIEW

File: pkg/pillar/cmd/ledmanager/ledmanager.go [hunk 1]
```diff
@@ -122,16 +122,16 @@ var mToF = []modelToFuncs{
 		// No dd disk light blinking on QEMU
 	},
 	{
-		model:     "raspberrypi,4-model-b.brcm,bcm2711",
+		model:     "raspberrypi.rpi.raspberrypi,4-model-b.brcm,bcm2711",
```
Comment (eriknordmark) [inline pkg/pillar/cmd/ledmanager/ledmanager.go:125]: Did something change in the device tree? Initially the model comes from /proc/device-tree/compatible and my RPI4 has raspberrypi,4-model-bbrcm,bcm2711

What do you have in that file?
I don't know if we need to match on each comma-sparated component?

Comment (rvs) [reply pkg/pillar/cmd/ledmanager/ledmanager.go:125]: Device tree is basically changing with every tweak of the kernel version @eriknordmark -- a better question is whether we have to latch on something more stable (like a subset of that string).

Comment (giggsoff) [reply pkg/pillar/cmd/ledmanager/ledmanager.go:125]: As I can see, there is a combination of lines, not the one from /proc/device-tree/compatible: https://github.com/lf-edge/eve/blob/master/pkg/pillar/hardware/model.go#L71.

Comment (eriknordmark) [reply pkg/pillar/cmd/ledmanager/ledmanager.go:125]: Sure, but the parts from dmidecode are empty on all ARM I've looked at. Does your RPi4 support dmidecode??

Comment (giggsoff) [reply pkg/pillar/cmd/ledmanager/ledmanager.go:125]: Yes. Here you are:
```# dmidecode 3.1
Getting SMBIOS data from sysfs.
SMBIOS 3.0 present.
7 structures occupying 253 bytes.
Table at 0x3CB26020.

Handle 0x0000, DMI type 0, 24 bytes
BIOS Information
	Vendor: U-Boot
	Version: 2020.07
	Release Date: 08/04/2020
	ROM Size: 64 kB
	Characteristics:
		PCI is supported
		BIOS is upgradeable
		Selectable boot is supported
		I2O boot is supported
		Targeted content distribution is supported

Handle 0x0001, DMI type 1, 27 bytes
System Information
	Manufacturer: raspberrypi
	Product Name: rpi
	Version: Not Specified
....
```

**Comment by rvs** [comment]
Pulling this in now for 5.8.2 testing -- @eriknordmark if you still have questions/comments lets address them in a separate PR if that's OK with you.

**Pull request merged**